src/CommandingOfficers/Strong.java

```diff
@@ -116,45 +119,58 @@ public void modifyUnitAttackOnUnit(BattleParams params)
     private static final int STRONGARM_COST = 4;
     private static final int STRONGARM_BUFF = 10;
     private static final int STRONGARM_FOOT_BUFF = 20; // On top of STONGARM_BUFF
+    UnitModifier damageMod = null;
+    UnitModifier defenseMod = null;
+    UnitTypeFilter damageModTroop = null;
+    UnitTypeFilter moveMod = null;
+
+    StrongArmAbility(Strong strong)
+    {
+      super(strong, STRONGARM_NAME, STRONGARM_COST);
 
-    CODamageModifier damageMod = null;
-    CODefenseModifier defenseMod = null;
-    CODamageModifier damageModTroop = null;
+      damageMod = new UnitDamageModifier(STRONGARM_BUFF);
+      defenseMod = new UnitDefenseModifier(STRONGARM_BUFF);
+      damageModTroop = new UnitTypeFilter(new UnitDamageModifier(STRONGARM_FOOT_BUFF));
+      damageModTroop.allOf = UnitModel.TROOP;
+
+      // Grant troops and transports additional movement power.
+      moveMod = new UnitTypeFilter(new UnitMovementModifier(2));
+      moveMod.oneOf = UnitModel.TROOP | UnitModel.TRANSPORT;
+    }
 
-    StrongArmAbility(Commander commander)
+    @Override
+    protected void enqueueUnitMods(MapMaster gameMap, ArrayList<UnitModifier> modList)
     {
-      super(commander, STRONGARM_NAME, STRONGARM_COST);
+      modList.add(damageMod);
+      modList.add(defenseMod);
+      modList.add(damageModTroop);
+      modList.add(moveMod);
+    }
 
-      damageMod = new CODamageModifier(STRONGARM_BUFF);
-      defenseMod = new CODefenseModifier(STRONGARM_BUFF);
-      damageModTroop = new CODamageModifier(STRONGARM_FOOT_BUFF);
-      for( UnitModel model : commander.getAllModels(UnitModel.TROOP) )
+    private HashMap<Commander, UnitProductionModifier> upmsApplied = new HashMap<>();
```

Comment:
Is there any reason to have this be a map instead of just the modifier?

Reply:
There was until we had those architecture discussions in person.

Reply:
So should we simplify this then?

Reply:
Done.